ui/app/mirrors/create/handlers.ts

```diff
@@ -66,6 +70,10 @@ const CDCCheck = (
   config.tableMappings = tableNameMapping as TableMapping[];
   config.flowJobName = flowJobName;
 
+  if (IsEventhubsPeer(destinationType)) {
```

Comment:
if only used once no need to make it a function

Reply:
it's fine, more readable